Packs/Redmine/Integrations/Redmine/Redmine.py

```diff
@@ -0,0 +1,603 @@
+
+
+def adjust_paging_to_request(args: dict[str, Any]):
+    page_number = args.pop('page_number', None)
+    page_size = args.pop('page_size', None)
+    limit = args.pop('limit', None)
+    offset_to_dict = None
+    limit_to_dict = None
+    if page_number or page_size:
+        if page_size:
+            page_size = int(page_size)
+        else:
+            page_size = 50
+        if page_number:
+            page_number = int(page_number)
+        else:
+            page_number = 1
+        offset_to_dict = (page_number - 1) * page_size
+        limit_to_dict = page_size
+        page_number_for_header = page_number
+    else:
+        if limit:
+            offset_to_dict = 0
+            limit_to_dict = int(limit) if int(limit) <= 100 else 100
+        else:
+            offset_to_dict = 0
+            limit_to_dict = 25
+        page_number_for_header = 1
+    return offset_to_dict, limit_to_dict, page_number_for_header
+
+
+def map_header(header_string: str) -> str:
+    header_mapping = {
+        'id': 'ID',
+        'author': 'Author',
+        'project': 'Project',
+        'status': 'Status',
+        'priority': ' Priority',
+        'login': 'Login',
+        'admin': 'Admin',
+        'firstname': 'First Name',
+        'lastname': 'Last Name',
+        'mail': 'Email',
+        'created_on': 'Created On',
+        'last_login_on': 'Last Login On',
+        'estimated_hours': 'Estimated Hours',
+        'start_date': 'Start Date',
+        'custom_fields': 'Custom Fields',
+        'description': 'Description',
+        'subject': 'Subject',
+    }
+    return header_mapping.get(header_string, header_string)
+
+
+def map_predefined_values_to_id(predefined_value, converter_dict, error_message):
+    if predefined_value is not None:
+        predefined_id = converter_dict.get(predefined_value)
+        if predefined_id is not None:
+            return predefined_id
+        else:
+            raise DemistoException(error_message)
+    return None
+
+
+def convert_args_to_request_format(args):  # put pop in line, remove exceptions
+    tracker_id = args.pop('tracker_id', None)
+    status_id = args.pop('status_id', None)
+    priority_id = args.pop('priority_id', None)
+    custom_fields = args.pop('custom_fields', None)
+    watcher_user_ids = args.pop('watcher_user_ids', None)
+
+    args['tracker_id'] = map_predefined_values_to_id(
+        tracker_id, ISSUE_TRACKER_DICT, "Tracker_id invalid, please make sure you use only predefined values")
+    args['status_id'] = map_predefined_values_to_id(
+        status_id, ISSUE_STATUS_DICT, "Status_id invalid, please make sure you use only predefined values")
+    args['priority_id'] = map_predefined_values_to_id(
+        priority_id, ISSUE_PRIORITY_DICT, "Priority_id invalid, please make sure you use only predefined values")
+
+    if custom_fields:
+        custom_fields = custom_fields.split(',')
```

Comment:
use ArgToList function 
Please go over other places and replace the split with this function if needed